docs/sources/reference/commandline/cli.md

```diff
@@ -169,6 +169,7 @@ expect an integer, and they can only be specified once.
       -l, --log-level="info"                 Set the logging level
       --label=[]                             Set key=value labels to the daemon
       --log-driver="json-file"               Default driver for container logs
+	  --log-opt=[]							 Log driver specific options
```

Comment:
Looks like you used tabs in this file in stead of spaces, or at least, the lines don't align properly. (Same for the other changes below)